sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -472,6 +472,27 @@ def test_KNeighborsRegressor_multioutput_uniform_weight():
         assert_array_almost_equal(y_pred, y_pred_idx)
 
 
+def test_radius_neighbors_regressor_zero_distance():
```

Comment:
Can you maybe check RadiusNeighborsRegressor and KNeighborsRegressor using a for loop similar to the other test?